root/static/scripts/common/components/ArtistRoles.js

```diff
@@ -24,14 +25,11 @@ type Props = {|
 
 const ArtistRoles = ({relations}: Props) => (
   <ul>
-    {relations.map(r =>(
+    {relations.map(r => (
```

Comment:
This was an accident but actually it seems to be more consistent with how we usually do it, so I left it in!